doc/admin/deployment.md

```diff
@@ -856,12 +857,13 @@ DAOS Agent is a standalone application to be run on each compute node.
 It can be configured to use secure communications (default) or can be allowed
 to communicate with the control plane over unencrypted channels. The following
 example shows daos_agent being configured to operate in insecure mode due to
-incomplete integration of certificate support as of the 0.6 release.
+incomplete integration of certificate support as of the 0.6 release and
+configured to use a non-default agent configuration file.
 
 To start the DAOS Agent from the command line, run:
 
 ```bash
-$ daos_agent -i
+$ daos_agent -i -o <'path to agent configuration file/daos_agent.yml'> &
```

Comment:
I might have two examples here. one for it not using -o to say that it will use /etc/daos/daos_agent.yml  (although this may be broken until we backport a patch) and a second fro using a config.